hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/BlobRenameHandler.java

```diff
@@ -519,12 +527,13 @@ private PathInformation getPathInformation(Path path,
 
       return new PathInformation(true,
           abfsClient.checkIsDir(op.getResult()),
-          extractEtagHeader(op.getResult()));
+          extractEtagHeader(op.getResult()),
+          op.getResult() instanceof AbfsHttpOperation.AbfsHttpOperationWithFixedResultForGetFileStatus);
```

Comment:
didnt get the need for instance check here

Reply:
getPathStatus on blobClient will give instance of AbfsHttpOperationWithFixedResultForGetFileStatus for an implicit path. Hence used.